plugins/LayerView/LayerView.py

```diff
@@ -392,10 +389,14 @@ def _updateWithPreferences(self):
 
         self.setLayerViewType(int(float(Preferences.getInstance().getValue("layerview/layer_view_type"))));
 
-        self.setExtruderOpacity(0, float(Preferences.getInstance().getValue("layerview/extruder0_opacity")))
-        self.setExtruderOpacity(1, float(Preferences.getInstance().getValue("layerview/extruder1_opacity")))
-        self.setExtruderOpacity(2, float(Preferences.getInstance().getValue("layerview/extruder2_opacity")))
-        self.setExtruderOpacity(3, float(Preferences.getInstance().getValue("layerview/extruder3_opacity")))
+        extruder_nr = 0
+        for extruder_opacity in Preferences.getInstance().getValue("layerview/extruder_opacities").split(","):
```

Comment:
Could we use a different separator? We have run into issues using , as separator before and it is awfully close to localized formatting of floats.